src/pages/ReimbursementAccount/ACHContractStep.js

```diff
@@ -36,74 +38,111 @@ const propTypes = {
 class ACHContractStep extends React.Component {
     constructor(props) {
         super(props);
+        this.validate = this.validate.bind(this);
 
         this.addBeneficialOwner = this.addBeneficialOwner.bind(this);
         this.submit = this.submit.bind(this);
 
         this.state = {
             ownsMoreThan25Percent: ReimbursementAccountUtils.getDefaultStateForField(props, 'ownsMoreThan25Percent', false),
             hasOtherBeneficialOwners: ReimbursementAccountUtils.getDefaultStateForField(props, 'hasOtherBeneficialOwners', false),
-            acceptTermsAndConditions: ReimbursementAccountUtils.getDefaultStateForField(props, 'acceptTermsAndConditions', false),
-            certifyTrueInformation: ReimbursementAccountUtils.getDefaultStateForField(props, 'certifyTrueInformation', false),
             beneficialOwners: ReimbursementAccountUtils.getDefaultStateForField(props, 'beneficialOwners', []),
         };
-
-        // These fields need to be filled out in order to submit the form (doesn't include IdentityForm fields)
-        this.requiredFields = [
-            'acceptTermsAndConditions',
-            'certifyTrueInformation',
-        ];
-
-        // Map a field to the key of the error's translation
-        this.errorTranslationKeys = {
-            acceptTermsAndConditions: 'common.error.acceptedTerms',
-            certifyTrueInformation: 'beneficialOwnersStep.error.certify',
-        };
-
-        this.getErrors = () => ReimbursementAccountUtils.getErrors(this.props);
-        this.clearError = inputKey => ReimbursementAccountUtils.clearError(this.props, inputKey);
-        this.clearErrors = inputKeys => ReimbursementAccountUtils.clearErrors(this.props, inputKeys);
-        this.getErrorText = inputKey => ReimbursementAccountUtils.getErrorText(this.props, this.errorTranslationKeys, inputKey);
     }
 
     /**
-     * @returns {Boolean}
+     * @param {Object} values
+     * @returns {Object}
      */
-    validate() {
-        let beneficialOwnersErrors = [];
+    validate(values) {
+        const errors = {};
+
         if (this.state.hasOtherBeneficialOwners) {
-            beneficialOwnersErrors = _.map(this.state.beneficialOwners, ValidationUtils.validateIdentity);
+            _.each(this.state.beneficialOwners, (ownerID) => {
+                // Don't validate ownerIDs that were removed from the beneficialOwners array
+                if (!_.contains(this.state.beneficialOwners, ownerID)) {
+                    return;
+                }
```

Comment:
I think this check might not be necessary. We are iterating over `this.state.beneficialOwners` so `ownerID` should always be contained in the array, right?